string app
string name
---
int8 result